core/src/test/resources/sql/cast-with-format.iq

```diff
@@ -979,7 +939,7 @@ EXPR$0
 select cast(date'2019-01-07' as varchar
     FORMAT 'WW');
 EXPR$0
-01
+02
```

Comment:
what is happening here?

Reply:
The `WW` operator returns the `week number of the year (00-53)`, and that depends on what is being set as the start day of a week (Sun v/s Mon). The variances you see here are because we set the week start on Sunday, but setting to Monday also fails a bunch of other tests, which is strange as you would expect all of them to be consistent. So we have to use either of them and update the tests accordingly.

Reply:
Since this is BigQuery, they better give the same results.